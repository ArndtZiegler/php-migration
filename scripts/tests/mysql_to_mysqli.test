$qid = mysql_query($sql, $db);