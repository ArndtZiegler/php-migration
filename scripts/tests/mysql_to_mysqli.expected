$qid = mysqli_query($db, $sql);